codestyle.xml

```diff
@@ -0,0 +1,23 @@
+<code_scheme name="spinnaker">
```

Comment:
No - you still need to highlight the code you want and apply the reformat command.